packages/headless/src/features/search-parameters/search-parameter-analytics-actions.test.ts

```diff
@@ -1,203 +1,13 @@
-    expectIdenticalActionType(
-      logParametersChange({fExcluded: {author: ['Cervantes']}}, {}),
-      logFacetUnexclude({facetId: 'author', facetValue: 'Cervantes'})
-    );
-  });
-
-  it('should log #logFacetClearAll when an fExcluded parameter with multiple values is removed', () => {
-    expectIdenticalActionType(
-      logParametersChange({fExcluded: {author: ['Cervantes', 'Orwell']}}, {}),
-      logFacetClearAll('author')
-    );
-  });
-
-  it('should log #logFacetUnexclude when an fExcluded parameter is modified & a value removed', () => {
-    expectIdenticalActionType(
-      logParametersChange(
-        {fExcluded: {author: ['Cervantes', 'Orwell']}},
-        {fExcluded: {author: ['Cervantes']}}
-      ),
-      logFacetUnexclude({facetId: 'author', facetValue: 'Orwell'})
-    );
-  });
-
-  it('should log #logFacetSelect when an fExcluded parameter is added', () => {
-    expectIdenticalActionType(
-      legacyLogParametersChange({}, {fExcluded: {author: ['Cervantes']}}),
-      logFacetExclude({facetId: 'author', facetValue: 'Cervantes'})
-    );
-  });
-
-  it('should log #logFacetSelect when an fExcluded parameter is modified & a value added', () => {
-    expectIdenticalActionType(
-      legacyLogParametersChange(
-        {fExcluded: {author: ['Cervantes']}},
-        {fExcluded: {author: ['Cervantes', 'Orwell']}}
-      ),
-      logFacetExclude({facetId: 'author', facetValue: 'Orwell'})
-    );
-  });
-}
-
-function legacyTestFacetLogging(
-  parameter: string,
-  expectIdenticalActionType: (
-    action1: LegacySearchAction,
-    action2: LegacySearchAction
-  ) => void
-) {
-  it(`should log #logFacetDeselect when an ${parameter} parameter with a single value is removed`, () => {
-    expectIdenticalActionType(
-      legacyLogParametersChange({[parameter]: {author: ['Cervantes']}}, {}),
-      logFacetDeselect({facetId: 'author', facetValue: 'Cervantes'})
-    );
-  });
-
-  it(`should log #logFacetClearAll when an ${parameter} parameter with multiple values is removed`, () => {
-    expectIdenticalActionType(
-      legacyLogParametersChange(
-        {[parameter]: {author: ['Cervantes', 'Orwell']}},
-        {}
-      ),
-      logFacetClearAll('author')
-    );
-  });
-
-  it(`should log #logFacetDeselect when an ${parameter} parameter is modified & a value removed`, () => {
-    expectIdenticalActionType(
-      legacyLogParametersChange(
-        {[parameter]: {author: ['Cervantes', 'Orwell']}},
-        {[parameter]: {author: ['Cervantes']}}
-      ),
-      logFacetDeselect({facetId: 'author', facetValue: 'Orwell'})
-    );
-  });
-}
+import {searchboxSubmit} from '../query/query-analytics-actions';
+import {resultsSort} from '../sort-criteria/sort-criteria-analytics-actions';
+import {parametersChange} from './search-parameter-analytics-actions';
 
-// --------------------- KIT-2859 : Everything above this will get deleted ! :) ---------------------
```

Comment:
Instead of migrating -> Just delete following the comment in the file.

Reply:
We drop testing for the legacy stuff before dropping the legacy stuff are we fine with that 🤔
(Honest question)

Reply:
Hum yeah, will bring them back then. It's an easy migration after all.